### PR TITLE
Add Swedish translation

### PR DESCRIPTION
**Pull request by Sopor**

### CODE REVIEW

File: vfs.rar/resources/language/resource.language.sv_se/strings.po [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# Addon id: vfs.rar
+# Addon Provider: Team Kodi
+msgid ""
+msgstr ""
+"Project-Id-Version: KODI Main\n"
+"Report-Msgid-Bugs-To: https://github.com/xbmc/vfs.rar/issues/\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Kodi Translation Team\n"
+"Language-Team: English (United Kingdom) (http://www.transifex.com/projects/p/kodi-main/language/en_GB/)\n"
```
Comment (AlwinEsch) [inline vfs.rar/resources/language/resource.language.sv_se/strings.po]: There should be then:
```po
"Language-Team: Swedish (Sweden) (http://www.transifex.com/projects/p/kodi-main/language/sv_SE/)\n"
```

**Review by AlwinEsch — COMMENTED** [review]
Can you also then look for here:
https://github.com/xbmc/vfs.rar/blob/Leia/vfs.rar/addon.xml.in#L18-L24

**EDIT:** Also is on `<disclaimer>` the `lang="en_GB"` missing.

Normally this translation is done by https://www.transifex.com/teamxbmc/kodi-main/content/ but the addon is there not included.

From my side OK to add, only must ask the Team to have accepted without Transifex.

Thanks a lot to bring this!

**Comment by AlwinEsch** [comment]
Reminder about the change on comment and to add on addon.xml.in.

**Comment by Sopor** [comment]
I have been trying to join on transifex but i can't find Swedish in the menu :(

**Comment by AlwinEsch** [comment]
Much addons are still missing in transifex, hope there comes time to add them there.

Can you update the request after the others here are in (have text changes).
Also was still a translated text from addon.xml.in missing (thats why still open and has also forgot the request)

**Comment by Sopor** [comment]
> Much addons are still missing in transifex, hope there comes time to add them there.
> 
> Can you update the request after the others here are in (have text changes).
> Also was still a translated text from addon.xml.in missing (thats why still open and has also forgot the request)

PR for the `addon.xml.in` file https://github.com/xbmc/vfs.rar/pull/93

**Comment by Sopor** [comment]
Doesn't this PR also need to be merged?

**Comment by AlwinEsch** [comment]
Thought to wait until the other, as there some new strings added.
Maybe you can then direct include your swedish about :smile:, if OK for you.

**Comment by AlwinEsch** [comment]
Can you update to the latest language changes and then I bring in this too.

**Comment by Sopor** [comment]
I have moved the Swedish translations here https://kodi.weblate.cloud so i assume i can close this PR ...

**Pull request closed without merging**